src/components/DatePicker.vue

```diff
@@ -130,6 +132,10 @@
     components: { Day },
 
     props: {
+      borderDateColor:{
+        default:null,
+        type:String,
```

Comment:
We could probably set the type to Object, and use the default style that we want in the `deafult` here instead of setting it to `null`. This way, the user can have total control of the current date appearance, not only its border color. 
like:

```
type: Object,
default: () => ({ border: "1px solid #00c690" }),
```